src/common/fs/FileUtils.cpp

```diff
@@ -488,24 +466,28 @@ void FileUtils::Iterator::fileNext() {
     }
 }
 
-
 void FileUtils::Iterator::openFileOrDirectory() {
     type_ = FileUtils::fileType(path_.c_str());
     if (type_ == FileType::DIRECTORY) {
         if ((dir_ = ::opendir(path_.c_str())) == nullptr) {
-            status_ = Status::Error("opendir `%s': %s", path_.c_str(), ::strerror(errno));
+            status_ = Status::Error("opendir `%s': %s",
+                                    path_.c_str(),
+                                    ::strerror(errno));
             return;
         }
     } else if (type_ == FileType::REGULAR) {
         fstream_ = std::make_unique<std::ifstream>();
         fstream_->open(path_);
         if (!fstream_->is_open()) {
-            status_ = Status::Error("open `%s': %s", path_.c_str(), ::strerror(errno));
+            status_ = Status::Error("open `%s': %s",
```

Comment:
Such changes are OK to me. But for your reference, our linting rule on column width is 100c, which you could check on `src/.linters`.

Reply:
I undo the change related to the linter, just focus on the logic part and repushed.